docs/event-bus/03-04-knative-eventing-mesh.md

```diff
@@ -41,3 +41,76 @@ To reach the new Eventing Mesh, use an HTTP request with the `/events` path.
 For example, if you have used `gateway.example.cx/v1/events` so far, use `gateway.example.cx/events` to make sure you work with the new Eventing Mesh. 
 
 >**NOTE:** The HTTP source adapter only accepts Events compliant with the [CloudEvents 1.0 specification](https://github.com/cloudevents/spec/blob/v1.0/spec.md).
+
+## Other Channel options
+
+By default Kyma comes with Natss but Knative eventing allows to exchange the default channel implementation. It even allows to have multiple channel implementations simultaneously.
+The default channel implementation can be changed during installation using an installation-override like this:
```

Comment:
```suggestion
You can [override](root/kyma/#configuration-helm-overrides-for-kyma-installation) the default channel implementation during installation like this:
```
I would make it more direct - shorten the sentence, use the active voice (we usually go for active voice whenever possible), and just write `override` directly. I also added a link, in case somebody doesn't know how to override stuff.

Reply:
A leading slash is missing for `root`. I will pick your suggestion and change it afterwards